CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v6.3.1](https://github.com/voxpupuli/puppet-zabbix/tree/v6.3.1) (2018-03-29)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-zabbix/compare/v6.3.0...v6.3.1)
+
+**Fixed bugs:**
+
+- listenip in agent not resolv to IP-address from network face name [\#493](https://github.com/voxpupuli/puppet-zabbix/issues/493)
```

Comment:
This issue title could probably be changed to make a bit more sense, but otherwise good to get a fixed release out as soon as possible.

Reply:
* changed issue title
* regenerated the CHANGELOG.md